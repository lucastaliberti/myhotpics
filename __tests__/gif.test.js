import renderer from 'react-test-renderer'
import Gif from '../components/gif'

const MOCK = {
  url: 'https://gif.myhot.pics/Chase%20Nordengren%20Nordo%20Pow.gif',
  title: 'Chase Nordengren Nordo Pow'
}

describe('component - gif', () => {
  it('should render', () => {
    const component = renderer.create(<Gif {...MOCK} />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
