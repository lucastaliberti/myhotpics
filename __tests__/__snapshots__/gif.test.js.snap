// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component - gif should render 1`] = `
<a
  className="bg-center cover dib relative w-50 ma0 fl"
  href="https://gif.myhot.pics/Chase%20Nordengren%20Nordo%20Pow.gif"
  style={
    Object {
      "backgroundImage": "url(https://gif.myhot.pics/Chase%20Nordengren%20Nordo%20Pow.gif)",
      "height": "0",
      "paddingBottom": "33.33333%",
    }
  }
  title="Chase Nordengren Nordo Pow"
/>
`;
